README.md

```diff
@@ -111,42 +111,53 @@ To install:
  
 ### CMake Options
 
-If you run the `cmake` command by hand, there are many options you can
-set with "-D". For example, the option to build the RADOS Gateway is
-defaulted to ON. To build without the RADOS Gateway:
+The `-D` flag can be used with the `cmake` command to speed up the process of
```

Comment:
I would write "can be used with `cmake` to"

Reply:
I like that better.